### PR TITLE
Fix(contacted people loading)

### PR DESCRIPTION
**Pull request by daniel7764**
contacted people loading is now conditional

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx [hunk 1]
```diff
@@ -29,10 +29,10 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
     const [currentInteractedContact, setCurrentInteractedContact] = React.useState<InteractedContact>();
     const [familyRelationships, setFamilyRelationships] = React.useState<FamilyRelationship[]>([]);
 
-    const { saveContactQuestioning, saveContact, updateInteractedContact, changeIdentificationType, openAccordion, updateNoResponse, loadInteractedContacts } = useContactQuestioning({ interactedContactsState, setCurrentInteractedContact });
+    const { saveContactQuestioning, saveContact, updateInteractedContact, changeIdentificationType, openAccordion, updateNoResponse, getMinimalDateToInvestigate } = useContactQuestioning({ interactedContactsState, setCurrentInteractedContact });
 
     React.useEffect(() => {
-        loadInteractedContacts();
+        getMinimalDateToInvestigate();
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx:35]: I think you should find a better name for this function. My first instinct is to think it returns minimal date to investigate but in reality it loads interacted c;ontacts by minimal date.

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 2]
```diff
@@ -1,40 +1,62 @@
...
+    const getMinimalDateToInvestigate = () => {
+        axios.get(`/clinicalDetails/coronaTestDate/${epidemiologyNumber}`).then((res: any) => {
+            if (res.data !== null) {
+                const coronaTestDate = convertDate(res.data.coronaTestDate);
+                const symptomsStartTime = convertDate(res.data.symptomsStartTime);
+                const doesHaveSymptoms = res.data.doesHaveSymptoms;
+                let earliestDate: Date;
+                if (coronaTestDate !== null) {
+                    if (doesHaveSymptoms !== null && doesHaveSymptoms) {
+                        if (symptomsStartTime)
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:45]: Add curly braces to if else blocks

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts (hunk 2 above)
Comment (daniel7764) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx (hunk 1 above)
Comment (daniel7764) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 3]
```diff
@@ -1,40 +1,63 @@
...
+                if (coronaTestDate !== null) {
+                    if (doesHaveSymptoms !== null && doesHaveSymptoms) {
+                        if (symptomsStartTime) {
+                            earliestDate = subDays(symptomsStartTime, symptomsWithKnownStartDate);
+                        } else {
+                            earliestDate = subDays(coronaTestDate, symptomsWithUnknownStartDate)
+                        }
+                    } else {
+                        earliestDate = subDays(coronaTestDate, nonSymptomaticPatient)
+                    }
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:52]: maybe export to func?

**Review by NaamanTsur — DISMISSED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts (hunk 3 above)
Comment (daniel7764) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts]: fixed

**Review by yaniryos — APPROVED** [review]

**Review by roymaniv — APPROVED** [review]

**Pull request merged**